parse/compose.go

```diff
@@ -7,6 +7,22 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+func CatalogInfoFromTemplateVersion(contents []byte) (model.Version, error) {
+
+	var data map[string]interface{}
```

Comment:
Didn't notice this was still here. Its leftover thoughts that I should separate into a different PR. Removing it.